### PR TITLE
wiki for feed management

### PR DESCRIPTION
**Pull request by ew3361zh**
**What**
Starting wiki document for the Feed Management module

**Why**
Development of full wiki coverage for entire RuFaS model

**How**
Based off [psuedocode documentation for Feed Management module](https://docs.google.com/document/d/1WxZXidjQLXTmBra_p2Pl0mi3fkj7RowJs3g-WIQKgU0/edit#heading=h.jke07n3yp5ok) and notes from/conversations with Kristan Reed.

Notes:
- Feed Management module is in development. Making sure this accurately describes current function and distinguishes between future enhancement functionality.

### CODE REVIEW

File: feed_management.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+## Feed Management and Inventory Model
+
+The **Feed Management and Inventory Model** is currently managed by the **Feed** class 
+and its subclass, **Storage**.
```
Comment (morrowcj) [inline feed_management.md]: In our docs, we've been formatting actual references to the code as in-line code blocks. For example, we might have done:

```suggestion
The **Feed Management and Inventory Model** is currently managed by the `Feed` class 
and its subclass, `Storage`.
```

File: feed_management.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+for the feeds managed by the farm and the methods for storage. It contains the 
+**Storage** subclass which specifies a single feed storage receptacle.
+
+Upon harvest, farm grown feed is grouped by crop type, harvest date, and quality, 
+then assigned to an available storage receptacle sequentially.
+
+Quality assessment for farm grown feeds is currently based on the fr_PHU 
+(fraction of potential heat units) accumulated by the crop at harvest and 
+occurs in the **Yields** class. Forage quality of “purchased” feeds is a database 
+entry queried in the **Feed** class.
```
Comment (morrowcj) [inline feed_management.md]: This is not currently true, but there are issues associated with it: #161 and #237

**Review by morrowcj — APPROVED** [review]
I think this looks great! I had a couple of minor comments but I especially like the "Future enhancements" section.

As a group, we're going to have to stay committed to updating these documents at each stage of development to keep them up-to-date.

File: feed_management.md (hunk 2 above)
Comment (ew3361zh) [reply feed_management.md]: What do you think about adding that as a future enhancement instead?

And do you think it is helpful to summarize the issues in the wiki as the "current" state of quality assessment?

**Review by KFosterReed — APPROVED** [review]
This looks like a great start! I added a few more sentences about the feed library. There is more to add as we improve this module but it is a great foundation to build from :)

File: feed_management.md (hunk 2 above)
Comment (morrowcj) [reply feed_management.md]: I think both are good ideas.

**Comment by ew3361zh** [comment]
Added page to GitHub wiki 02/01/2023

**Pull request closed without merging**